<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoilParameter</class>
 <widget class="QDialog" name="SoilParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Input Soil Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vertical hydraulic conductivity kv (m/s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ratio kh/kv</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ratio ks/kh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Void ratio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Horizontal consolidation coefficient Ch(m2/s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Bulk modulus (kN/m2)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Poisson's Ratio</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="kvLine">
         <property name="text">
          <string>1e-9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ratioKhLine">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ratioKsLine">
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="voidRatioLine">
         <property name="text">
          <string>1.8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ChLine">
         <property name="text">
          <string>5e-7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="KLine"/>
       </item>
       <item>
        <widget class="QLineEdit" name="PoissonRatioLine">
         <property name="text">
          <string>0.2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>kh: Horizontal hydraulic conductivity (m/s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ks: Horizontal hydraulic conductivity of the smear zone (m/s)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
